main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/PacketFire/paste-click/handlers/health"
+	"github.com/PacketFire/paste-click/handlers/upload"
+
+	"net/http"
+
+	cs "github.com/PacketFire/paste-click/lib/config/service"
+	"github.com/PacketFire/paste-click/lib/middleware/logging"
+	"github.com/PacketFire/paste-click/lib/objectstore"
+	"github.com/PacketFire/paste-click/lib/objectstore/drivers/local"
+	"github.com/PacketFire/paste-click/lib/objectstore/drivers/mock"
+
+	"github.com/gorilla/mux"
+)
+
+// store takes a string representing drivers and attempts to return a
+// corresponding driver if there is no match, nil is returned.
+func store(driverName string) objectstore.ObjectStore {
+	switch driverName {
+	case `local`:
```

Comment:
Would it be better to call this `fs` or `filesystem`, since you could point it to a NFS mounted directory if you wanted?

Reply:
I agree, this has been renamed to `fs`.